third_party/blink/renderer/bindings/core/v8/local_window_proxy.cc

```diff
@@ -159,6 +160,20 @@ void LocalWindowProxy::DisposeContext(Lifecycle next_status,
   lifecycle_ = next_status;
 }
 
+static const char* RecordReplayGetProcessType(
+  bool isMainWorld, bool isRoot, bool isDevtools) {
```

Comment:
can this function be changed to take the World / Frame and do the checks here instead of passing 3 bools?e